
A official kde-3.2 + qt-3.3 adaption patch sent by:

  - Raphael Langerhorst <raphael-langerhorst@gmx.at> 

--- kdelibs-3.1.95/kdefx/kstyle.cpp.orig	2004-01-17 13:53:01.000000000 +0100
+++ kdelibs-3.1.95/kdefx/kstyle.cpp	2004-01-20 08:37:44.625573288 +0100
@@ -971,14 +971,6 @@
 		case PM_MaximumDragDistance:
 			return -1;
 
-#if QT_VERSION >= 0x030300
-		case PM_MenuBarItemSpacing:
-			return 3;
-
-		case PM_ToolBarItemSpacing:
-			return 0;
-#endif
-
 		default:
 			return QCommonStyle::pixelMetric( m, widget );
 	}
